plugins/Kaleidoscope-DynamicMacros/src/kaleidoscope/plugin/DynamicMacros.cpp

```diff
@@ -176,7 +190,7 @@ void DynamicMacros::play(uint8_t macro_id) {
     case MACRO_ACTION_STEP_TAP_SEQUENCE: {
       while (true) {
         key.setFlags(0);
-        key.setKeyCode(pgm_read_byte(pos++));
+        key.setKeyCode(Runtime.storage().read(pos++));
```

Comment:
You're absolutely right.  I actually meant to move this change to a separate PR, because it's only related in that I discovered it by accident (IWYU failed because of it, though I'm still not sure why it fails, but other builds succeeded).